.storybook-test/config.js

```diff
@@ -1,23 +1,14 @@
-import {configure, addParameters} from '@storybook/react';
-import styleProcessor from 'wix-style-processor';
+import {configure} from '@storybook/react';
 import './stories.scss';
+import { init } from '../test/visual/StyleProcessorUtil';
 
 function loadStories() {
   require('../mocks');
-  styleProcessor.init();
 
   const req = require.context('../src', true, /\.visual\.tsx$/);
   req.keys().forEach(filename => req(filename));
-}
-
-configure(loadStories, module);
 
-addParameters({
-  options: {
-    name: 'wix-ui-tpa',
-    url: 'https://github.com/wix-private/wix-ui-tpa',
-    showAddonPanel: false,
-  }
-});
+  setTimeout(init);
```

Comment:
@shlomitc 
I put the `styleProcessor.init()` call here, in the `config` file.
However I have to add a timeout for this to work.
Not the `VisualContainerElement` only needs to wait for the promise to resolve, and the `init` is called once.
Wdyt?